Tasks/Common/codecoverage-tools/cobertura/cobertura.gradle.ccenabler.ts

```diff
@@ -0,0 +1,63 @@
+/// <reference path="../../../../definitions/Q.d.ts" />
+/// <reference path="../../../../definitions/string.d.ts" />
+/// <reference path="../../../../definitions/vsts-task-lib.d.ts" />
+/// <reference path="../../../../definitions/node.d.ts" />
+
+import * as util from "../utilities";
+import * as tl from "vsts-task-lib/task";
+import * as ccc from "../codecoverageconstants";
+import * as cc from "../codecoverageenabler";
+import * as str from "string";
+import * as os from "os";
+import * as Q from "q";
+
+export class CoberturaGradleCodeCoverageEnabler extends cc.CoberturaCodeCoverageEnabler {
+    // -----------------------------------------------------
+    // Enable code coverage for Cobertura Gradle Builds
+    // - enableCodeCoverage: CodeCoverageProperties  - ccProps
+    // -----------------------------------------------------    
+    public enableCodeCoverage(ccProps: { [name: string]: string }): Q.Promise<boolean> {
+        let _this = this;
+
+        _this.buildFile = ccProps["buildfile"];
+        let classFilter = ccProps["classfilter"];
+        let isMultiModule = ccProps["ismultimodule"] && ccProps["ismultimodule"] === "true";
+        let classFileDirs = ccProps["classfilesdirectories"];
+        let reportDir = ccProps["reportdirectory"];
+        let codeCoveragePluginData = null;
```

Comment:
consider declaring the type of variable you're using, e.g. let codeCoveragePluginData: string = null;